common/src/main/java/com/lowdragmc/lowdraglib/gui/widget/SliderWidget.java

```diff
@@ -0,0 +1,189 @@
+package com.lowdragmc.lowdraglib.gui.widget;
+
+import com.lowdragmc.lowdraglib.gui.editor.annotation.Configurable;
+import com.lowdragmc.lowdraglib.gui.editor.annotation.LDLRegister;
+import com.lowdragmc.lowdraglib.gui.editor.annotation.NumberRange;
+import com.lowdragmc.lowdraglib.gui.editor.configurator.IConfigurableWidget;
+import com.lowdragmc.lowdraglib.gui.texture.IGuiTexture;
+import com.lowdragmc.lowdraglib.gui.texture.ResourceBorderTexture;
+import com.lowdragmc.lowdraglib.utils.Position;
+import com.lowdragmc.lowdraglib.utils.Size;
+import dev.latvian.mods.kubejs.typings.Info;
+import dev.latvian.mods.rhino.util.RemapPrefixForJS;
+import lombok.Getter;
+import lombok.Setter;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.util.Mth;
+import org.jetbrains.annotations.NotNull;
+
+@LDLRegister(name = "slider", group = "widget.basic")
+@RemapPrefixForJS("kjs$")
+public class SliderWidget extends Widget implements IConfigurableWidget {
```

Comment:
Slider looks pretty good.
The main issue here is that it doesn;t support data sync between `server` and `remote`. so far all data can only be access from the client side